<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
    <line>
      <![CDATA[Test actual Started]]>
    </line>
    <line>
      <![CDATA[Test Started]]>
    </line>
    <line>
      <![CDATA[Test executed successfully]]>
    </line>
    <line>
      <![CDATA[Test Started]]>
    </line>
    <line>
      <![CDATA[This method failed and taken ss: createUser]]>
    </line>
    <line>
      <![CDATA[Test Started]]>
    </line>
    <line>
      <![CDATA[Test Skipped]]>
    </line>
    <line>
      <![CDATA[Test finished]]>
    </line>
  </reporter-output>
  <suite started-at="2023-07-26T15:14:39 IST" name="Default suite" finished-at="2023-07-26T15:14:50 IST" duration-ms="10203">
    <groups>
    </groups>
    <test started-at="2023-07-26T15:14:39 IST" name="Default test" finished-at="2023-07-26T15:14:50 IST" duration-ms="10203">
      <class name="testNGPackage.TestCase">
        <test-method is-config="true" signature="setUp()[pri:0, instance:testNGPackage.TestCase@4b29d1d2]" started-at="2023-07-26T15:14:40 IST" name="setUp" finished-at="2023-07-26T15:14:43 IST" duration-ms="3410" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginMethod()[pri:0, instance:testNGPackage.TestCase@4b29d1d2]" started-at="2023-07-26T15:14:43 IST" name="loginMethod" finished-at="2023-07-26T15:14:44 IST" duration-ms="738" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Test Started]]>
            </line>
            <line>
              <![CDATA[Test executed successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginMethod -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testNGPackage.TestCase@4b29d1d2]" started-at="2023-07-26T15:14:44 IST" name="tearDown" finished-at="2023-07-26T15:14:44 IST" duration-ms="483" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:testNGPackage.TestCase@4b29d1d2]" started-at="2023-07-26T15:14:44 IST" name="setUp" finished-at="2023-07-26T15:14:46 IST" duration-ms="2138" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="createUser()[pri:0, instance:testNGPackage.TestCase@4b29d1d2]" started-at="2023-07-26T15:14:46 IST" name="createUser" finished-at="2023-07-26T15:14:49 IST" depends-on-methods="testNGPackage.TestCase.loginMethod" duration-ms="2469" status="FAIL">
          <exception class="org.openqa.selenium.ElementClickInterceptedException">
            <message>
              <![CDATA[element click intercepted: Element <div class="label">...</div> is not clickable at point (677, 27). Other element would receive the click: <img src="/img/default/pixel.gif?hash=1692528820" height="98" width="84" class="sizer">
  (Session info: chrome=114.0.5735.199)
Build info: version: '4.9.0', revision: 'd7057100a6'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.6'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [e4522e73059e01250d4c4723cab0026e, clickElement {id=10207DCFE446401261798AE930642C7D_element_64}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 114.0.5735.199, chrome: {chromedriverVersion: 114.0.5735.16 (7e1ff058633f..., userDataDir: C:\Users\DELL\AppData\Local...}, goog:chromeOptions: {debuggerAddress: localhost:51007}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:51007/devtoo..., se:cdpVersion: 114.0.5735.199, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on WINDOWS (e4522e73059e01250d4c4723cab0026e)] -> xpath: //div[text()='Users']]
Session ID: e4522e73059e01250d4c4723cab0026e]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <div class="label">...</div> is not clickable at point (677, 27). Other element would receive the click: <img src="/img/default/pixel.gif?hash=1692528820" height="98" width="84" class="sizer">
  (Session info: chrome=114.0.5735.199)
Build info: version: '4.9.0', revision: 'd7057100a6'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.6'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [e4522e73059e01250d4c4723cab0026e, clickElement {id=10207DCFE446401261798AE930642C7D_element_64}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 114.0.5735.199, chrome: {chromedriverVersion: 114.0.5735.16 (7e1ff058633f..., userDataDir: C:\Users\DELL\AppData\Local...}, goog:chromeOptions: {debuggerAddress: localhost:51007}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:51007/devtoo..., se:cdpVersion: 114.0.5735.199, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on WINDOWS (e4522e73059e01250d4c4723cab0026e)] -> xpath: //div[text()='Users']]
Session ID: e4522e73059e01250d4c4723cab0026e
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:193)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:183)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:158)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:257)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:78)
at testNGPackage.TestCase.createUser(TestCase.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementClickInterceptedException -->
          <reporter-output>
            <line>
              <![CDATA[Test Started]]>
            </line>
            <line>
              <![CDATA[This method failed and taken ss: createUser]]>
            </line>
          </reporter-output>
        </test-method> <!-- createUser -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testNGPackage.TestCase@4b29d1d2]" started-at="2023-07-26T15:14:49 IST" name="tearDown" finished-at="2023-07-26T15:14:50 IST" duration-ms="463" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="logout()[pri:0, instance:testNGPackage.TestCase@4b29d1d2]" started-at="2023-07-26T15:14:50 IST" name="logout" finished-at="2023-07-26T15:14:50 IST" depends-on-methods="testNGPackage.TestCase.createUser" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method testNGPackage.TestCase.logout() on instance testNGPackage.TestCase@4b29d1d2 depends on not successfully finished methods 
[testNGPackage.TestCase.createUser() on instance testNGPackage.TestCase@4b29d1d2]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method testNGPackage.TestCase.logout() on instance testNGPackage.TestCase@4b29d1d2 depends on not successfully finished methods 
[testNGPackage.TestCase.createUser() on instance testNGPackage.TestCase@4b29d1d2]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
            <line>
              <![CDATA[Test Skipped]]>
            </line>
            <line>
              <![CDATA[Test finished]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- testNGPackage.TestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
